src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -180,6 +181,14 @@ private void start() throws Exception {
 		this.context.start();
 		this.onReplicatorStart();
 		replicator.runLoop();
+
+		// Register the replication lag metric here because there are codepaths that allow for creating multiple
+		// replicators (at least in the tests).
+		// Doing so results in attempting to create multiple metrics with the same name which causes an exception.
+		MaxwellMetrics.registry.register(
+			MetricRegistry.name(Maxwell.class, "replication", "lag"),
```

Comment:
I feel all the metrics name should be defined as constant in MaxwellMetrics class.